pyanaconda/kickstart.py

```diff
@@ -1961,28 +1960,7 @@ def is_harddrive_based(self):
         return self.partition is not None
 
 class ReqPart(COMMANDS.ReqPart):
-    def execute(self, storage, ksdata):
-        if not self.reqpart:
-            return
-
-        log.debug("Looking for platform-specific bootloader requirements.")
-        reqs = platform.set_platform_bootloader_reqs()
-
-        if self.addBoot:
-            log.debug("Looking for platform-specific boot requirements.")
-            bootPartitions = platform.set_platform_boot_partition()
-
-            # Blivet doesn't know this - anaconda sets up the default boot fstype
-            # in various places in this file. We need to duplicate that here.
-            for part in bootPartitions:
-                if part.mountpoint == "/boot":
-                    part.fstype = storage.default_boot_fstype
-
-            reqs += bootPartitions
-
-        if reqs:
-            log.debug("Applying requirements:\n%s", "".join(map(str, reqs)))
-            autopart.do_reqpart(storage, reqs)
+    pass
```

Comment:
Do we need this class when there is no change in the implementation?

Reply:
Fixed.